package-lock.json

```diff
@@ -463,7 +463,7 @@
     },
     "ansi-escapes": {
       "version": "3.1.0",
-      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
```

Comment:
Not sure why this happens, but it looks like npm bug https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285/39